src/assets/i18n/messages.de.json

```
@@ -0,0 +1,198 @@
+{
+  "17f0ea5d2d7a262b0e875acc70475f102aee84e6": "Playlist erstellen",
+  "cff1428d10d59d14e45edec3c735a27b5482db59": "Name",
+  "f47e2d56dd8a145b2e9599da9730c049d52962a2": "Audiodateien",
+  "a52dae09be10ca3a65da918533ced3d3f4992238": "Videos",
+  "d9e83ac17026e70ef6e9c0f3240a3b2450367f40": "Youtube-dl Argumente ändern",
+  "7fc1946abe2b40f60059c6cd19975d677095fd19": "Simulierte neue Argumente",
+  "0b71824ae71972f236039bed43f8d2323e8fd570": "Argument hinzufügen",
+  "c8b0e59eb491f2ac7505f0fbab747062e6b32b23": "Nach Kategorie filtern",
+  "9eeb91caef5a50256dd87e1c4b7b3e8216479377": "Argument-Wert verwenden",
+  "25d8ad5eba2ec24e68295a27d6a4bb9b49e3dacd": "Argument-Wert",
+  "7de2451ed3fb8d8b847979bd3f0c740b970f167b": "Argument hinzufügen",
+  "d7b35c384aecd25a516200d6921836374613dfe7": "Abbrechen",
+  "b2623aee44b70c9a4ba1fce16c8a593b0a4c7974": "Ändern",
+  "038ebcb2a89155d90c24fa1c17bfe83dbadc3c20": "YouTube Downloader",
+  "6d2ec8898344c8955542b0542c942038ef28bb80": "Bitte geben Sie eine gültige URL ein.",
+  "a38ae1082fec79ba1f379978337385a539a28e73": "Qualität",
+  "4be966a9dcfbc9b54dfcc604b831c0289f847fa4": "URL verwenden",
+  "d3f02f845e62cebd75fde451ab8479d2a8ad784d": "Ansehen",
+  "4a9889d36910edc8323d7bab60858ab3da6d91df": "Nur Audio",
+  "96a01fafe135afc58b0f8071a4ab00234495ce18": "Multi-Download Modus",
+  "6a21ba5fb0ac804a525bf9ab168038c3ee88e661": "Download",
+  "6a3777f913cf3f288664f0632b9f24794fdcc24e": "Abbrechen",
+  "322ed150e02666fe2259c5b4614eac7066f4ffa0": "Erweitert",
+  "b7ffe7c6586d6f3f18a9246806a7c7d5538ab43e": "Simulierter Befehl:",
+  "4e4c721129466be9c3862294dc40241b64045998": "Benutzerdefinierte Argumente verwenden",
+  "ad2f8ac8b7de7945b80c8e424484da94e597125f": "Benutzerdefinierte Argumente",
+  "a6911c2157f1b775284bbe9654ce5eb30cf45d7f": "Der FQDN muss nicht angegeben werden, sondern nur der Teil danach. Argumente werden wie folgt getrennt: ,,",
+  "3a92a3443c65a52f37ca7efb8f453b35dbefbf29": "Benutzerdefinierte Ausgabe verwenden",
+  "d9c02face477f2f9cdaae318ccee5f89856851fb": "Benutzerdefinierte Ausgabe",
+  "fcfd4675b4c90f08d18d3abede9a9a4dff4cfdc7": "Dokumentation",
+  "19d1ae64d94d28a29b2c57ae8671aace906b5401": "Der Pfad ist relativ zum Konfigurations-Download-Pfad. Dateiendung nicht angeben.",
```

```suggestion
  "19d1ae64d94d28a29b2c57ae8671aace906b5401": "Der Pfad ist relativ zum Konfigurations-Download-Pfad. Dateiendung nicht mit angeben.",
```
Not sure about this one.

---

This is not necessary imo. Let me know when you think differently.

---

"Dateiendung auslassen" wäre auch eine Möglichkeit? :)